pymeasure/instruments/keithley/keithley2450.py

```diff
@@ -635,3 +635,4 @@ def shutdown(self):
             self.ramp_to_voltage(0.0)
         self.stop_buffer()
         self.disable_source()
+        super().shutdown()
```

Comment:
Log message could be removed

Reply:
This is interesting -- the log message is useful as it tells the user that shutdown has been *initiated*.

As the log message in `Instrument.shutdown` comes at the end of that method, when everything is finished and nothing more is expected to happen, we should change the text of _that_ message to `log.info(f"Finished shutting down {self.name)}")`